stock_serial_no_auto_reserve/models/stock_move.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2024 Sunflower IT (http://sunflowerweb.nl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    is_available = fields.Boolean(
+        "Is available but not assigned", compute="_compute_is_available", store=True
+    )
```

Comment:
So I think the reason why the installation of the module is slow, and why you needed the init script, is because of this computed field.

Some points to note:

- It doesn't make sense to have stored computed fields in Odoo at all, if you don't specify an `api.depends` clause for them. What Odoo will do in the background for such fields is, for each of the `api.depends` members, it creates a trigger on `write`, and it calls the computation function when triggered. Currently, you don't have any such triggers defined, so the computation will never trigger. The only two cases where this field is being written to is 1. when you write to it manually, which you're doing in `_action_assign` below, and when you first install the module and the field is created.

- My point then being, why does this need to be a stored computed field at all? Just execute the code that does the computation on `action_assign`, that's the only trigger we need. You can then also dispense with the init script, because no computation will run anymore upon install (only the one for pickings, but that one should be fast enough)